contracts/AelinDeal.sol

```diff
@@ -77,7 +76,7 @@ contract AelinDeal is AelinVestingToken, MinimalProxyFactory, IAelinDeal {
          * calculates the amount of underlying deal tokens you get per wrapped deal token accepted
          */
         underlyingPerDealExchangeRate = (_dealData.underlyingDealTokenTotal * 1e18) / maxTotalSupply;
-        emit SetHolder(_dealData.holder);
+        emit HolderSet(_dealData.holder);
```

Comment:
@0xcdb I'm updating subgraph to conform these changes and little bit confused, shouldn't we emit `HolderAccepted` here?

Reply:
good one @0xDmitry thanks